.nais/deploy-veiledning-catbox.yml

```diff
@@ -60,3 +60,5 @@ spec:
   env:
     - name: "PENSJONSKALKULATOR_BACKEND"
       value: "http://pensjonskalkulator-backend"
+    - name: "AZURE_OBO_ISSUER"
```

Comment:
Jeg ville kanskje ha kalt denne `ENTRA_ID_OBO_SCOPE`, da det er snakk om hvilket scope som Entra ID (tidligere Azure AD) skal utstede token for, så "issuer" blir litt misvisende her.